src/test/java/io/xpring/codec/addresses/XAddressCodecTest.java

```diff
@@ -0,0 +1,65 @@
+package io.xpring.codec.addresses;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.xpring.xrpl.ClassicAddress;
+import org.junit.Test;
+
+public class XAddressCodecTest {
+
+  @Test
+  public void testEncodeMainnetNoTag() {
```

Comment:
optional: Prefer using GIVEN / WHEN / THEN which we've tried to adopt as a standard across Xpring SDK and Xpring's repos in general (https://martinfowler.com/bliki/GivenWhenThen.html). I think it makes tests easier to grok quickly and more readable.